credentials/src/main/kotlin/web5/credentials/VerifiableCredential.kt

```diff
@@ -0,0 +1,250 @@
+package web5.credentials
+
+import com.danubetech.verifiablecredentials.CredentialSubject
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.KotlinModule
+import com.nfeld.jsonpathkt.JsonPath
+import com.nfeld.jsonpathkt.extension.read
+import com.nimbusds.jose.JWSAlgorithm
+import com.nimbusds.jose.JWSHeader
+import com.nimbusds.jose.jwk.JWK
+import com.nimbusds.jose.util.Base64URL
+import com.nimbusds.jwt.JWTClaimsSet
+import com.nimbusds.jwt.JWTParser
+import com.nimbusds.jwt.SignedJWT
+import foundation.identity.did.DIDURL
+import foundation.identity.did.VerificationMethod
+import web5.sdk.common.Convert
+import web5.sdk.crypto.Crypto
+import web5.sdk.dids.Did
+import web5.sdk.dids.DidResolvers
+import java.net.URI
+import java.security.SignatureException
+
+/**
+ * Type alias representing the danubetech Verifiable Credential data model.
+ * This typealias simplifies the use of the [com.danubetech.verifiablecredentials.VerifiableCredential] class.
+ */
+public typealias VcDataModel = com.danubetech.verifiablecredentials.VerifiableCredential
```

Comment:
I know VerifiableCredentialType was a weak name, but VcDataModel hmm..

Just brainstorming..
VerifiableCredentialDataModel
VerifiableCredentialModel 
VerifiableCredentialCore
VerifiableCredentialEntity

Reply:
Also kind of strange that you moved these out of the model folder, once you add the SSITest.kt you will find you need a lot of the sub parts of VC exported like CredentialSubject